src/WebJobs.Script/Workers/Rpc/IWorkerConfigurationResolver.cs

```diff
@@ -1,20 +1,18 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the MIT License. See License.txt in the project root for license information.
 
 using System.Collections.Generic;
-using Microsoft.Azure.WebJobs.Script.Workers.Rpc.Configuration;
-using Microsoft.Extensions.Options;
 
 namespace Microsoft.Azure.WebJobs.Script.Workers.Rpc
 {
     /// <summary>
-    /// Provides an abstraction for retrieving worker configuration resolution details.
+    /// Provides an abstraction for retrieving worker configurations.
     /// </summary>
     internal interface IWorkerConfigurationResolver
     {
         /// <summary>
-        /// Retrieves the worker configuration resolution information which includes the root directory path of workers and worker configuration paths.
+        /// Retrieves a dictionary of worker configurations, keyed by language name.
         /// </summary>
-        WorkerConfigurationInfo GetConfigurationInfo();
+        Dictionary<string, RpcWorkerConfig> GetWorkerConfigs();
```

Comment:
```suggestion
        IReadOnlyDictionary<string, RpcWorkerConfig> GetWorkerConfigs();
```

Is there any need for callers to mutate this dictionary?